test/index.js

```diff
@@ -234,4 +234,31 @@ describe('Hexo Renderer Markdown-it', () => {
       result.should.equal('<h2 id="foo">foo<a class="anchor" href="#foo">#</a></h2>\n');
     });
   });
+
+  describe('nunjucks', () => {
+    const hexo = new Hexo(__dirname, { silent: true });
+    const loremFn = () => { return 'ipsum'; };
+    const engine = 'md';
+
+    before(async () => {
+      await hexo.init();
+      hexo.extend.tag.register('lorem', loremFn);
+      hexo.extend.renderer.register('md', 'html', require('../lib/renderer'));
```

Comment:
should be sync mode, will update after https://github.com/hexojs/hexo/pull/4498 released.

PS: hexo-renderer-marked should be [updated](https://github.com/hexojs/hexo-renderer-marked/blob/646dad4a66dcec6ac957ccce6e24e5889ce04bf0/test/index.js#L872) too.

Reply:
https://github.com/hexojs/hexo-renderer-marked/pull/171